### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by DaniLatreCodes**
Hi Bilal this my new pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#HELLO WORLD
+
+##Intro
+
+
```
Comment (bilalelbedoui93) [inline README.md]: @DaniLatreCodes "intro no", but "introduction yes"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#HELLO WORLD
+
```
Comment (bilalelbedoui93) [inline README.md]: @DaniLatreCodes finish your TODO please

File: README.md (hunk 1 above)
Comment (DaniLatreCodes) [reply README.md]: @bilalelbedoui93  Sorry but intro is OK

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+##Intro
+
+
+
+##Functional description
+
+lorem ipsum
+
```
Comment (bilalelbedoui93) [inline README.md]: @DaniLatreCodes it's not enough, there are some lines missing

**Pull request merged**